<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <item>
        <property name="text">
         <string>test1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>test2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>test3</string>
        </property>
       </item>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>536</width>
          <height>744</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>240</width>
               <height>426</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>426</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>11</number>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <row>
              <property name="text">
               <string>Название фильма</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Оригинальное название</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Год</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Страна</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Жанр</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Длительность</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Рейтинг</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Режиссёр</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Сценарий</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Актёры</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Описание</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Test</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <row>
            <property name="text">
             <string>Имя файла</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Размер</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Разрешение</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Видео кодек</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Битрейт</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Продолжительность</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Аудио</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Субтитры</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Test</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTest_Action"/>
  </widget>
  <action name="actionTest_Action">
   <property name="text">
    <string>Test Action</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
